.gitignore

```diff
@@ -3,3 +3,4 @@ env
 venv
 *.sqlite3
 .idea
+database/
```

Comment:
Czy to jest tutaj potrzebne ? :D

Reply:
Postgres przy tej konfiguracji stworzy top-level folder database, jak chcemy żeby go nie wrzuciło do gita, to raczej tak. CHYBA że chcemy podpiąć folder progresa w innymi miejscu

Reply:
Oki, nie widziałem volume w Docker, super jest :)